tutorials/iot/ubuntu-kiosk/electron-kiosk.md

```diff
@@ -340,8 +330,7 @@ We now have the .snap file on the device in its home directory. We need to insta
 
 ```bash
 snap install --dangerous ./electron-hello-world-kiosk_0.1_arm64.snap 
-snap connect electron-hello-world-kiosk:browser-sandbox :browser-support
-snap connect electron-hello-world-kiosk:x11-plug electron-hello-world-kiosk:x11
+snap connect electron-hello-world-kiosk:wayland
 snap restart electron-hello-world-kiosk
```

Comment:
`:wayland` got autoconnected here (fresh Core VM) and the kiosk app just started, so the two lines above can go away AFAICT.

Reply:
Fixed